redash/query_runner/clickhouse.py

```diff
@@ -74,7 +74,7 @@ def _send_query(self, data, stream=False):
             timeout=self.configuration.get('timeout', 30),
             params={
                 'user': self.configuration['user'],
-                'password':  self.configuration['password'],
+                'password':  self.configuration.get('password', ""),
                 'database': self.configuration['dbname']
```

Comment:
Maybe we will use `.get('dbname', 'default')`?

Reply:
Well, at the moment `dbname` is a required field, so it's guaranteed we will have a value for it.

Reply:
I didn't want to leave the same comment on 2 lines. :)

I think that dbname param should be not required if we agree that user and password fields could be not required too.

Reply:
👍 got it. I'm going to merge this as is. But I don't mind making `dbname` optional. I just don't want to throw more work at @aidarbek :)